tests/unit/utils/test_verify.py

```diff
@@ -10,10 +10,15 @@
 import sys
 import stat
 import shutil
-import resource
 import tempfile
 import socket
 
+# Import third party libs
+try:
```

Comment:
I'm wondering if it would be better here to be more explicit and just do this gating behind a `sys.platform` check instead of just using the side-effect of a library being available or not.